src/Traits/ComparisonTrait.php

```diff
@@ -240,6 +240,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
+     *
+     * @return bool
+     */
+    public function isNextWeek()
+    {
+        return $this->toDateString() === static::now($this->tz)->addWeek()->toDateString();
```

Comment:
Also, this method seems incomplete or the description in the method is misleading. What does it mean to be "within" next week? This method seems to return true only if i is exactly next week.